src/components/EjectButton/EjectButton.test.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import React from 'react';
+import { shallow } from 'enzyme';
+import { remote } from 'electron'; // Mocked
+import EjectButton, { dialogOptions, dialogCallback } from './EjectButton';
+import BigClickableButton from '../BigClickableButton';
+
+const { dialog } = remote;
+
+describe('EjectButton component', () => {
+  let wrapper;
+  let clickHandler;
+
+  beforeEach(() => {
+    clickHandler = jest.fn();
+    wrapper = shallow(<EjectButton onClick={clickHandler} />);
+  });
+
+  it('should render (not running)', () => {
```

Comment:
At first glance it was a little unclear to me what running/not running meant, until I realized it was the pressed state... I think it could be changed to something like:

```
it('should render without being pressed (not running)', () => {

it('should render as pressed (running)', () => {
```

Reply:
Good catch. I modified it.